src/nameserver/system_table.h

```diff
@@ -163,20 +163,16 @@ class SystemTable {
                 break;
             }
             case SystemTableType::kUser: {
-                SetColumnDesc("host", type::DataType::kString, table_info->add_column_desc());
-                SetColumnDesc("user", type::DataType::kString, table_info->add_column_desc());
-                SetColumnDesc("password", type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("Host", type::DataType::kString, table_info->add_column_desc());
```

Comment:
Is there any usage over this table from the previous release ? If so it would a breaking change

Reply:
Yes it's a breaking change